src/test/java/com/vladbudan/app/service/impl/UserServiceImplTest.java

```diff
@@ -0,0 +1,147 @@
+package com.vladbudan.app.service.impl;
+
+import com.vladbudan.app.dto.converter.UserConverter;
+import com.vladbudan.app.dto.model.UserDto;
+import com.vladbudan.app.model.User;
+import com.vladbudan.app.repository.UserRepository;
+import com.vladbudan.app.service.UserService;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+
+public class UserServiceImplTest {
+
+    private UserService userService;
+
+    @Mock
+    private UserConverter userConverter;
+
+    @Mock
+    private UserRepository userRepository;
+
+    @Before
+    public void init() {
+        initMocks(this);
+        userService = new UserServiceImpl(userRepository, userConverter);
+    }
+
+    @Test
+    public void getUsersTest() {
+
+        List<User> list = new ArrayList<>();
```

Comment:
Please read about given/when/then convention. How to correctly comment test classes.
https://blog.codecentric.de/en/2017/09/given-when-then-in-junit-tests/